examples/MFLFDTesting.ipynb

```diff
@@ -0,0 +1,1345 @@
+{
```

Comment:
Should this file be here?

Reply:
No. It should not. Sorry about that. Will remove!